cmd/service.go

```diff
@@ -136,8 +136,11 @@ func serviceRestartCmd() *cobra.Command {
 	cmdconfig.
 		OnCmd(cmd).
 		AddBoolFlag(constants.ArgHelp, "h", false, "Help for service restart").
-		AddBoolFlag(constants.ArgForce, "", false, "Forces the service to restart, releasing all open connections and ports")
-
+		AddBoolFlag(constants.ArgForce, "", false, "Forces the service to restart, releasing all open connections and ports").
+		// dashboard server
+		AddBoolFlag(constants.ArgDashboard, "", false, "Run the dashboard webserver with the service").
+		AddStringFlag(constants.ArgDashboardListen, "", string(dashboardserver.ListenTypeNetwork), "Accept connections from: local (localhost only) or network (open) (dashboard)").
```

Comment:
I don't think `ArgDashboardPort` and `ArgDashboardListen` should be exposed as args - we should just reuse values for the currently running dashboard service (are these in the dashboard state?)

Reply:
makes sense. fixed